app/views/onboarding/EnableExposureNotifications.tsx

```diff
@@ -28,55 +25,53 @@ export const EnableExposureNotifications = (): JSX.Element => {
   const titleText = t('label.launch_exposure_notif_header');
 
   const handleOnPressEnable = () => {
-    requestExposureNotificationAuthorization();
+    requestENAuthorization();
     dispatchOnboardingComplete();
   };
 
   return (
-    <Theme use='violet'>
```

Comment:
Removed the Theme component as we are no longer using multiple themes in the app, no other changes were done to the mark up.